.github/actions/setup-environment/action.yml

```diff
@@ -4,15 +4,15 @@ runs:
   using: "composite"
   steps:
 
-  - name: 'Set up JDK 8'
+  - name: 'Set up JDK 17'
     uses: actions/setup-java@v4
     with:
       distribution: 'temurin'
-      java-version: 8
+      java-version: 17
 
-  - name: Prepare JDK8 env var
+  - name: Prepare JDK17 env var
```

Comment:
🤔 Maybe add single quotes to all the names of the steps here.

Reply:
I don't think we generally do that. How would it help?

Reply:
It seems that about half of the steps use single quotes. (like the one in line 7: `name: 'Set up JDK 17'`. But it's surely not important at all.

Reply:
Hah, I did not pay attention to that, sorry. I think it makes sense to align it. Fixed it in [`6996a67` (#429)](https://github.com/gradle/test-retry-gradle-plugin/pull/429/commits/6996a67b7736bde8c30cae363621faaf207bd366)